### DISCUSSION TITLE
请问，如何设置库级别审批流程？

### DISCUSSION THREAD

**Discussion by cyfanhu** [hhyo/Archery#1962] [Q&A]
目前1个数据库集群有多个数据库schema，有没有办法设置每个schema拥有自己的owner并配置在审批？

**Comment by nick2wang** [comment]
变通方案是将一个实例拆分成多个实例配置，每个分实例的数据库账号只授权特定的数据库权限，相当于每个schema/数据库可以独立设置资源组和审批流程配置
